data/Makefile.am

```diff
@@ -1,9 +1,9 @@
 libwacomtabletdir = $(datadir)/libwacom
-tablet_files = $(shell find $(top_srcdir)/data -name "*.tablet" -printf "%P\n")
+tablet_files = $(shell cd $(top_srcdir)/data; printf '%s\n' *.tablet)
```

Comment:
Ideally, `cd foo; printf '%s\n' *.ext` could be replaced by `basename foo/*.ext` but GNU version of `basename` doesn't support more than one argument.